src/startup.jl

```diff
@@ -44,8 +44,19 @@ if !symbols_present
     isfile(depfile) || error("PyCall not properly installed. Please run Pkg.build(\"PyCall\")")
     include(depfile) # generated by Pkg.build("PyCall")
     # Only to be used at top-level - pointer will be invalid after reload
-    libpy_handle = Libdl.dlopen(libpython, Libdl.RTLD_LAZY|Libdl.RTLD_DEEPBIND|Libdl.RTLD_GLOBAL)
+    try
+        Libdl.dlopen(libpython, Libdl.RTLD_LAZY|Libdl.RTLD_DEEPBIND|Libdl.RTLD_GLOBAL)
+    catch err
+        if err isa ErrorException
+            error(err.msg, ". Please run `Pkg.build(\"PyCall\")` if your Python build has changed")
+        else
+            error("Please run `Pkg.build(\"PyCall\")` if your Python build has changed")
```

Comment:
No `error` here.  Just `rethrow`.

Reply:
(If the libpython file is missing, it will throw an `ErrorException`.)

Reply:
Ok, removed. But, if the only two possibilities are an `ErrorException` object or a `Ptr` to nothing (if the file is found and the `catch` block is not executed) then, do we even need the `if`?

Reply:
I don't know that those are the only two possibilities.  I only know that if the exception is not an `ErrorException` then the problem is probably not the Python installation.